packages/aws/data_stream/config/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,422 @@
+      field: json.ComplianceType
+      tag: rename_ComplianceType
+      target_field: aws.config.compliance_type
+      ignore_missing: true
+  - set:
+      field: result.evaluation
+      tag: set_result_evaluation_from_aws_config_compliance_type
+      copy_from: aws.config.compliance_type
+      ignore_empty_value: true
+  - script:
+      tag: set_event_outcome_from_aws_config_compliance_type
+      lang: painless
+      description: set event.outcome from aws.config.compliance_type
+      if : ctx.aws?.config?.compliance_type instanceof String
+      source: >-
+        if (ctx.aws.config.compliance_type == 'NON_COMPLIANT') {
+          ctx.event.outcome = 'failure';
+        } else if (ctx.aws.config.compliance_type == 'COMPLIANT') {
+          ctx.event.outcome = 'success';
+        }else {
+          ctx.event.outcome = 'unknown';
+        }
+  - rename:
+      field: json.ConfigRuleInfo.Scope.TagValue
+      tag: rename_ConfigRuleInfo_Scope_TagValue
+      target_field: aws.config.rule_info.scope.tag_value
+      ignore_missing: true
+  - set:
+      field: rule.tags
+      tag: set_rule_tags_from_config_config_rule_info_scope_tag_value
+      copy_from: aws.config.rule_info.scope.tag_value
+      ignore_empty_value: true  
+  - remove:
+      field:
+        - aws.config.evaluation_result_identifier.ordering_timestamp
+        - aws.config.evaluation_result_identifier.resource_evaluation_id
+        - aws.config.rule_info.config_rule_arn
+        - aws.config.rule_info.config_rule_id
+        - aws.config.rule_info.config_rule_name
+        - aws.config.rule_info.description
+      tag: remove_custom_duplicate_fields
+      ignore_missing: true
+      if: ctx.tags == null || !ctx.tags.contains('preserve_duplicate_custom_fields')
+  - remove:
+      field: json
+      tag: remove_json
+      ignore_missing: true
+  - script:
+      description: This script processor iterates over the whole document to remove fields with null values.
+      tag: script_to_drop_null_values
+      lang: painless
+      source: |
+        void handleMap(Map map) {
+          map.values().removeIf(v -> {
+            if (v instanceof Map) {
+                handleMap(v);
+            } else if (v instanceof List) {
+                handleList(v);
+            }
+            return v == null || v == '' || (v instanceof Map && v.size() == 0) || (v instanceof List && v.size() == 0)
+          });
+        }
+        void handleList(List list) {
+          list.removeIf(v -> {
+            if (v instanceof Map) {
+                handleMap(v);
+            } else if (v instanceof List) {
+                handleList(v);
+            }
+            return v == null || v == '' || (v instanceof Map && v.size() == 0) || (v instanceof List && v.size() == 0)
+          });
+        }
+        handleMap(ctx);
+  - set:
+      field: event.kind
+      tag: set_pipeline_error_into_event_kind
+      value: pipeline_error
+      if: ctx.error?.message != null
```

Also need the original event recovery in this case.
```suggestion
  - set:
      field: event.kind
      tag: set_pipeline_error_into_event_kind
      value: pipeline_error
      if: ctx.error?.message != null
  - append:
      field: tags
      value: preserve_original_event
      allow_duplicates: false
      if: ctx.error?.message != null
```